### PR TITLE
Regenerate CRDs/APIs with openapi-gen

### PR DESCRIPTION
**Pull request by dustman9000**
Regenerate CRDs/APIs using boilerplate's openapi-gen such that validate fill succeed.

This will get https://github.com/openshift/release/pull/16830 to run successfully.

### CODE REVIEW

**Comment by 2uasimojo** [comment]
Need to rename deploy/crds/{cloudingress.managed.openshift.io_apischemes_crd.yaml => cloudingress.managed.openshift.io_APISchemes_crd.yaml}

File: deploy/crds/cloudingress.managed.openshift.io_apischemes_crd.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: apischemes.cloudingress.managed.openshift.io
+  name: APISchemes.cloudingress.managed.openshift.io
```
Comment (2uasimojo) [inline deploy/crds/cloudingress.managed.openshift.io_apischemes_crd.yaml:4]: This is odd. Is a resource allowed to have uppercase in its name? None of the CRDs on my stage cluster do.

Comment (2uasimojo) [reply deploy/crds/cloudingress.managed.openshift.io_apischemes_crd.yaml:4]: The [k8s docs](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/) imply that "most' object names can't contain uppercase letters. Even if CRDs are technically an execption, none of the other CRDs I'm looking at have uppercase in their names.

Comment (dustman9000) [reply deploy/crds/cloudingress.managed.openshift.io_apischemes_crd.yaml]: To get this back to lowercase, I needed to update the kubebuilder resource path from APISchems to apischems. Marking as resolved.

**Comment by dustman9000** [comment]
> Need to rename deploy/crds/{cloudingress.managed.openshift.io_apischemes_crd.yaml => cloudingress.managed.openshift.io_APISchemes_crd.yaml}

This is no longer needed as I fixed the kubebuilder resource path name in the apischemes_type.go.

**Comment by 2uasimojo** [comment]
Delta looks right and `container-make validate` passes for me.

/lgtm

Thanks for hammering at this, Dustin :)

**Pull request merged**